src/stories/Tabs.stories.js

```diff
@@ -3,18 +3,215 @@ import React from 'react';
 import { storiesOf } from '@storybook/react';
 import { action } from '@storybook/addon-actions';
 
-import { Tabs } from 'components';
+import { Tabs, Text } from 'components';
 
 storiesOf('Tabs', module).add('Tabs', () => (
```

Comment:
Is there a way to make the component not have the weird oval there if a number isn't passed along?

Reply:
Yep. I can add a new commit if needed. I thought it was an indicator of the active tab, so I didn't touch it.

Reply:
Yeah that would be awesome if you can fix that.